features/git-prune-branches/main_branch/stale_non_feature_branch/without_open_changes.feature

```diff
@@ -19,3 +19,14 @@ Feature: git prune-branches: don't remove stale non-feature branches when called
       | local      | main, production |
       | remote     | main, production |
       | coworker   | main             |
+
+
+  Scenario: undoing the operation
+    When I run `git prune-branches --undo`
+    Then I get the error "Cannot undo"
```

Comment:
Seeing a test with this now, I think this error should be updated to `Nothing to undo`